python/chronos/src/bigdl/chronos/model/tf2/Seq2Seq_keras.py

```diff
@@ -17,76 +17,126 @@
 import tensorflow as tf
 from tensorflow.keras import backend as K
 from bigdl.nano.tf.keras import Model
-from tensorflow.keras.layers import LSTM, Dense, Lambda, Reshape
+from tensorflow.keras.layers import LSTM, Dense, Lambda, Reshape, Layer
+
+
+class Encoder(Layer):
+    def __init__(self, input_feature_num,
+                 lstm_hidden_dim=128, lstm_layer_num=2, dropout=0.2):
+        self.input_feature_num = input_feature_num
+        self.lstm_hidden_dim = lstm_hidden_dim
+        self.lstm_layer_num = lstm_layer_num
+        self.dropout = dropout
+        self.encoder_lstm = []
+        for i in range(lstm_layer_num):
+            self.encoder_lstm.append(LSTM(self.lstm_hidden_dim,
+                                          return_sequences=True,
+                                          input_shape=(None, input_feature_num),
```

Comment:
this input_feature_num should be dynamic (i.e. the value for this parameter is different for 1st layer and 2nd layer